add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -1312,11 +1313,24 @@ this.MozLoopService = {
 
       let window = gWM.getMostRecentWindow("navigator:browser");
       if (window) {
+        // The participant that joined isn't necessarily included in room.participants (depending on
+        // when the broadcast happens) so concatenate.
+        let isOwnerInRoom = room.participants.concat(participant).some(p => p.owner);
+
+        let localizedString;
+        if (isOwnerInRoom) {
+          localizedString =
+            MozLoopServiceInternal.localizedStrings.get("rooms_room_joined_owner_connected_label2");
+        } else {
+          let l10nString = MozLoopServiceInternal.localizedStrings.get("rooms_room_joined_owner_not_connected_label");
```

Comment:
You wrap above, but not for this one :smile: 
I'd like to suggest the following:
 - Put `let bundle = MozLoopServiceInternal.localizedStrings;` next to `isOwnerInRoom`.
 - Replace occurrences of `MozLoopServiceInternal.localizedStrings` in this function with `bundle`.